pysd/py_backend/xmile/SMILE2Py.py

```diff
@@ -125,7 +98,28 @@
     "mod": "%",
 }
 
-builders = {}
+# ====
+# 3.5.3 Delay Functions
+# http://docs.oasis-open.org/xmile/xmile/v1.0/csprd01/xmile-v1.0-csprd01.html#_Toc398039982
+# ====
+
+builders = {
+    # !TODO! Should correct handle the last args, should take the initial value of element instead the 0 if this parameter is missing
+    # "delay" !TODO! How to add the infinity delay?
```

Comment:
I don't support pipeline delays in the basic version of pysd. It either breaks the integration paradigm by requiring the model to go back and look at previous values of its integration (breaking the markov property) or requires that you have a delay order which is the delay time / integration step size, which is a pain. Nobody has complained yet, but if you think this is important, we can revisit. (It's going to be **slow** if we do try to implement it...)

Reply:
I'm not sure that's is real requred to implement, but this function is described in XMILE specification and I'm add it here. May be we should add some placeholder and raise some warning etc. 

Also I'm not fully understand what mean the infinity delay in XMILE specification. 
http://docs.oasis-open.org/xmile/xmile/v1.0/csprd01/xmile-v1.0-csprd01.html#_Toc398039982

Reply:
Lets open an issue to discuss that: https://github.com/JamesPHoughton/pysd/issues/147